docs/runtime.md

```diff
@@ -6,10 +6,18 @@ sidebar_position: 8
 
 Xpanse runtime is the running module built on SpringBoot.
 
+`Xpanse Runtime` has two startup methods, one is to use the local terraform module, the other is to
+use the remote terraform-boot service, the default is to use the local terraform
+
 ## Pre-Requisites
 
-1. If running the runtime jar directly, then the host must have `Terraform CLI` installed.
-2. Fully configured Zitadel instance.
+1. If using local terraform module, the `Xpanse Runtime` host must have `Terraform CLI` installed.
+
+2. If using remote terraform-boot service,A remote `Terrafom-boot` service must be deployed.
```

Comment:
If using remote terraform-boot service,A remote `Terrafom-boot` service must be deployed. => If using remote terraform-boot service, a remote `Terrafom-boot` service must be deployed/started.